blog/2022-07-27-app-platform-v10.md

```diff
@@ -0,0 +1,14 @@
+---
+slug: 2022/07/app-platform-v10
+title: App-platform v10
+author: ismay
+tags: [dhis2, app-platform]
+---
+
+We have just released a new major version of the app-platform: version 10. This new version should be relatively easy to update to for everyone. In this post we'll walk you through the most important change for this new version.
+
+## Node support
+
+The one breaking change for this release is that we've dropped support for node 12. This means that the app-platform libraries now only support the current LTS versions of node, which are node 14 and up. Node 12 is no longer under active development and we were seeing some of our dependencies dropping support for it, so we felt it was time we did the same.
```

Comment:
```suggestion
The one breaking change for this release is that we've dropped support for node 12. This means that now only the current LTS versions of nodejs will be supported by the app-platform libraries, which is node 14 and up. Node 12 is no longer under active development and we were seeing some of our dependencies dropping support for it, so we felt it was time we did the same.
```

Reply:
I have no idea if what I suggested is correct at all, just suggested what sounds "more correct" to me, please refer to a native speaker or simply ignore my suggestion